onnx/defs/math/defs.cc

```diff
@@ -538,24 +558,88 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyCelu)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* gelu_ver20_doc = R"DOC(
+Gelu takes one input data (Tensor<T>) and produces one
+output data (Tensor<T>) where the gaussian error linear units function, 
+`y = 0.5 * x * (1 + erf(x/sqrt(2)))` is applied to the tensor elementwise. 
+If the attribute "approximate" is set to "tanh", the function estimation,  
+`y = 0.5 * x * (1 + Tanh(sqrt(2/π) * (x + 0.044715 * x^3)))` is used and applied 
+to the tensor elementwise.
+
+)DOC";
+
+static std::string gelu_default_approx = "none";
+
+bool BuildContextDependentFunctionBodyGelu(
+    const FunctionBodyBuildContext& ctx,
+    const OpSchema& schema,
+    FunctionProto& functionProto) {
+  auto approx_attr_proto = ctx.getAttribute("approximate");
+  std::string approx  = approx_attr_proto != nullptr && approx_attr_proto->has_s()
+                      ? approx_attr_proto->s() 
+                      : gelu_default_approx;
+  FunctionBuilder builder(functionProto);
+  
+  if (approx == "tanh") {
```

Comment:
```suggestion
  if (approximate == "tanh") {
```

nit: I would spell it out and avoid abbreviations

Reply:
Done